python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -850,40 +916,48 @@ def mult(df):
                    a  b  c
                 0  1  1  1
                 2  2  1  3
+
+            ``engine='jit'`` can be used to accelerate certain functions,
+            initially those that contain reductions and arithmetic operations
+            between results of those reductions:
+
+            .. code-block::
```

Comment:
This docstring is formatted a little oddly. A few changes we should make:
1. Remove the `code-block` markup, that's implicit in everything starting with >>> in the Examples section.
2. The JIT example should be moved out of the pandas-compat block, it's not related to that
3. We should verify that doctests are being run for these examples, I'm not sure if they are with the current formatting.

Reply:
I think the doctests aren't running because `GroupBy` isn't part of `cudf.__all__`. I'll target a fix for this separately.

Reply:
OK that's fine with me.